indexer/services/roundtable/__tests__/tasks/create-pnl-ticks.test.ts

```diff
@@ -305,7 +312,7 @@ describe('create-pnl-ticks', () => {
       expect(pnlTicks.length).toEqual(1);
       expect(logger.info).toHaveBeenCalledWith(
         expect.objectContaining({
-          message: 'Skipping run because update interval has not been reached',
+          message: 'Skipping run because update interval has not been reached and all subaccounts have been processed',
```

Comment:
nit: create separate test to check the count condition

Reply:
done